src/components/Rocket/ProfileRockets.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export default function ProfileRockets() {
```

Comment:
- [ ] As required by project requirements, please kindly make use of React conditional rendering syntax in your Rockets and Missions.

### For Example:


  ```
 <span>
          {rockets.reserved && (
              <span className="inline-flex items-center justify-center px-2 py-1 mr-2 text-xs font-bold  text-white bg-[#18A2B8] rounded-md">Reserved</span>
              )}
            </span>
            <div className="mb-2">{rockets.description}</div>
            <div className="text-container">
              {rockets.reserved && (
              <button className="   bg-white p-1 text-gray-500 border-2 rounded-sm hover:bg-gray-200" 
                    type="button" onClick={() => dispatch(canRocket(rockets.id))}>
                {' '}
                Cancel Reservation
              </button>
              )}
              {' '}
              {!rockets.reserved && (
              <button className="rounded-sm px-2 p-1 bg-blue-500 text-white" type="button" onClick={() => 
                   dispatch(resRocket(rockets.id))}>
                {' '}
                Reserve Rocket
              </button>
```

The above is an example and may not apply directly to your code